src/Components/Film.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import InfoIcon from "@material-ui/icons/Info";
+import CloseIcon from "@material-ui/icons/Close";
+import AddIcon from "@material-ui/icons/Add";
+import RemoveIcon from "@material-ui/icons/RemoveRedEye";
+
+const Film = ({ film, addToWatchList, removeFromWatchList, clearList }) => {
+  const { l, q, i, s, y, rank, id } = film;
+
+  const [more, setMore] = useState(false);
+
+  const [added, setAdded] = useState(
+    localStorage.getItem("watchList")
```

Comment:
do you really need to get this from localeStorage here? or you can pass watchList by props?

Reply:
Here I just making sure that localstorage info updates immediately after components mout, for prevent some cases when info is not updated (I had some troubles with buttons "add to WL" and "remove from WL").
Of course I had possibility to pass Watchlist by props, but localstorage is available in any file and storing the freshest info. This is why I decided to update it. :)